pywinauto/application.py

```diff
@@ -1247,6 +1248,32 @@ def kill(self):
     # Non PEP-8 aliases
     kill_ = Kill_ = kill
 
+    def is_process_running(self):
+        """
+        Checks that process is still running
+
+        Returns True if process is still running otherwise - False
+        """

```

Comment:
Yes, why not? You meant here to update docstring?

I will add test later for this